package.json

```diff
@@ -81,6 +81,6 @@
     "raven": "^2.1.0",
     "speakeasy": "^2.0.0",
     "stream-file-archive": "^1.1.3",
-    "uws": "^9.14.0"
+    "express-uws": "^4.0.2"
```

Comment:
This isn't the same package as uws, and would likely break the websocket code

Reply:
‘’express-uws”: “^4.0.2.”